test/index.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+const RyderSerial = require("../out/index");
```

Comment:
I think we should have a test file at some point, however I am not a huge fan of this one because it cannot be automated / has hardcoded values. We probably need a way to deliver the simulator and run it in headless mode before unit tests make sense. What do you think?

Reply:
Agreed completely. Honestly only included this to showcase that it IS working without requiring you to mess around with any `package.lock` files in the `ryder-cli-proto` to confirm haha.

I will delete this folder.